cmd/acra-keys/keys/list-keys_test.go

```diff
@@ -88,13 +89,13 @@ testing     |        | 2023-02-14 23:40:28 +0000 UTC | Another ID
 func TestPrintKeysJSON(t *testing.T) {
 	keys := []keystore.KeyDescription{
 		{
-			ID:      "Another ID",
+			KeyID:   "Another KeyID",
 			Purpose: "testing",
 		},
 	}
 
 	output := bytes.Buffer{}
-	err := PrintKeys(keys, &output, &CommonKeyListingParameters{useJSON: true})
+	err := printKeysJSON(keys, &output)
```

Comment:
we should test json generation with rotated keys too to capture errors when they merged incorrectly